e2e/__snapshots__/automated.test.js.snap

```diff
@@ -1,18 +1,18 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`End to End tests diff view links commentbox 1`] = `27`;
+exports[`End to End tests diff view links commentbox 1`] = `28`;
```

Comment:
This is driving me nuts.

If I run `document.querySelectorAll('.octolinker-link').length` on https://github.com/OctoLinker/OctoLinker/pull/451/files#diff-1fb26bc12ac780c7ad7325730ed09fc4c2c3d757c276c3dacc44bfe20faf166f, it return 29 on this branch as well as with the latest main build. 

I have no idea why this was 27 in the first place.